internal/handler/service.go

```diff
@@ -13,4 +17,5 @@ type Service interface {
 	DeleteEmployee(id string) error
 	UpdatePosition(p *internal.Position) error
 	UpdateEmployee(e *internal.Employee) error
+	LogCorrelationID(r *http.Request) error
```

Comment:
This function is not a part of business logic of the service, so it shouldn't been a part of public interface. The function should be private and being called from the other Service functions.